<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapStylingDialog</class>
 <widget class="QDialog" name="MapStylingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ê³ ê³ í•™ ë„ë©´ ì‹œê°í™” ë„êµ¬ (Map Styler) - ArchToolkit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblHeader">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;ì „ë¬¸ê°€ìš© ì§€ë„ ìŠ¤íƒ€ì¼ë§&lt;/span&gt;&lt;/p&gt;&lt;p&gt;ë²¡í„° ë°ì´í„°(ë„ë¡œ, ìˆ˜ê³„, ê±´ë¬¼)ë¥¼ ê³ ê³ í•™ ë„ë©´ í‘œì¤€ìœ¼ë¡œ ì‹œê°í™”í•˜ê³ ,&lt;br/&gt;ë°°ê²½ ì§€í˜•(DEM)ì— ìŒì˜ê¸°ë³µ(Hillshade) íš¨ê³¼ë¥¼ ì ìš©í•©ë‹ˆë‹¤.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string>background-color: #fce4ec; padding: 10px; border-radius: 4px; border: 1px solid #f8bbd0;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupLayerSelect">
     <property name="title">
      <string>1. ìŠ¤íƒ€ì¼ì„ ì ìš©í•  ì›ë³¸ ë ˆì´ì–´ ì„ íƒ</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="lstLayers">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnSelectAll">
          <property name="text">
           <string>ëª¨ë‘ ì„ íƒ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDeselectAll">
          <property name="text">
           <string>ì„ íƒ í•´ì œ</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupVectorStyle">
     <property name="title">
      <string>2. ë²¡í„° ë°ì´í„° ìŠ¤íƒ€ì¼ë§ ì˜µì…˜</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="chkRoads">
        <property name="text">
         <string>ğŸ›£ï¸ ë„ë¡œ (Roads): ë“±ê¸‰ë³„(ê³ ì†/êµ­ë„/ì§€ë°©ë„) ì„ í­ ë° ìƒ‰ìƒ ì ìš©</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkRivers">
        <property name="text">
         <string>ğŸ’§ í•˜ì²œ/ìˆ˜ê³„ (Rivers): ìœ ì—­ ê·œëª¨ì— ë”°ë¥¸ ìœ„ê³„ í‘œí˜„</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkBuildings">
        <property name="text">
         <string>ğŸ  ê±´ë¬¼ (Buildings): ì…ì²´ê° ë¶€ì—¬ ë° ê·¸ë¦¼ì íš¨ê³¼</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_Shadow">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_Shadow">
          <property name="text">
           <string>â†³ ê±´ë¬¼ ê·¸ë¦¼ì ì˜¤í”„ì…‹ (ê¹Šì´):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spinOffset">
          <property name="value">
           <double>0.5</double>
          </property>
          <property name="singleStep">
           <double>0.1</double>
          </property>
          <property name="suffix">
           <string> mm</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
   <widget class="QGroupBox" name="groupDemStyle">
     <property name="title">
      <string>3. ë°°ê²½ ì§€í˜• (DEM) ì‹œê°í™”</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="chkDemStyling">
        <property name="text">
         <string>ğŸ”ï¸ ë°°ê²½ ì§€í˜• ìŠ¤íƒ€ì¼ë§ ì ìš© (ìŒì˜ê¸°ë³µ + ê³ ë„ì±„ìƒ‰)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QgsMapLayerComboBox" name="cmbDemLayer">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupPresetExport">
     <property name="title">
      <string>4. í”„ë¦¬ì…‹/ë‚´ë³´ë‚´ê¸°</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_PresetExport">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_CodeConfig">
        <item>
         <widget class="QLabel" name="label_CodeConfig">
          <property name="text">
           <string>DXF ì½”ë“œ ë§¤í•‘(JSON):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblCodeConfigPath">
          <property name="text">
           <string>(ìë™)</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_CodeButtons">
        <item>
         <widget class="QPushButton" name="btnOpenCodeConfig">
          <property name="text">
           <string>ë§¤í•‘ íŒŒì¼ ì—´ê¸°</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnReloadCodeConfig">
          <property name="text">
           <string>ë‹¤ì‹œ ë¶ˆëŸ¬ì˜¤ê¸°</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnExportPreset">
        <property name="text">
         <string>ğŸ“¦ QML/í”„ë¦¬ì…‹ ë‚´ë³´ë‚´ê¸°...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblPresetHint">
        <property name="text">
         <string>ì„ íƒí•œ ìŠ¤íƒ€ì¼(ë„ë¡œ/í•˜ì²œ/ê±´ë¬¼) QMLê³¼ í˜„ì¬ DXF ì½”ë“œ ë§¤í•‘ JSONì„ í´ë”ë¡œ ì €ì¥í•©ë‹ˆë‹¤. (DEM ìŠ¤íƒ€ì¼ì€ DEM ì„ íƒ+ì²´í¬ ì‹œ í•¨ê»˜ ì €ì¥)</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_Buttons">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnApply">
       <property name="text">
        <string>âœ¨ ìŠ¤íƒ€ì¼ ì ìš© (Apply Styling)</string>
       </property>
       <property name="styleSheet">
        <string>padding: 10px 20px; font-weight: bold; background-color: #ec407a; color: white; border-radius: 4px;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>ë‹«ê¸°</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
